privaterelay/glean/server_events.py

```diff
@@ -90,6 +90,37 @@ def emit_record(self, now: datetime, ping: dict[str, Any]) -> None:
 
         print(ping_envelope_serialized)
 
+    def record_api_accessed(
+        self,
+        user_agent: str,
+        ip_address: str,
+        endpoint: str,
+        method: str,
+        fxa_id: str,
+    ) -> None:
+        """
+        Record and submit a api_accessed event:
+        An API endpoint was accessed.
+        Event is logged to STDOUT via `print`.
+
+        :param str user_agent: The user agent.
+        :param str ip_address: The IP address. Will be used to decode Geo information
+            and scrubbed at ingestion.
+        :param str endpoint: The name of the endpoint accessed
+        :param str method: HTTP method used
+        :param str fxa_id: Mozilla accounts user ID
+        """
+        event = {
+            "category": "api",
+            "name": "accessed",
+            "extra": {
+                "endpoint": str(endpoint),
+                "method": str(method),
+                "fxa_id": str(fxa_id),
```

Comment:
I'm not sure why `glean_parser` wrapped these argument values in `str()` calls this time, but not in other places?